apps/admin/src/components/faq/FaqTable/FaqTable.tsx

```diff
@@ -4,16 +4,27 @@ import styled from 'styled-components';
 import FaqTableHeader from './FaqTableHeader/FaqTableHeader';
 import FaqTableItem from './FaqTableItem/FaqTableItem';
 
-const FaqTable = () => {
+interface Props {
+  selectedCategory: string;
+}
+
+const FaqTable = ({ selectedCategory }: Props) => {
   const { data: faqList } = useFaqListQuery();
+
+  const filteredFaqList =
+    selectedCategory === 'BOARD_ALL'
+      ? faqList
+      : faqList?.filter((item) => item.category === selectedCategory);
+
   return (
     <StyledFaqTable>
       <FaqTableHeader />
-      {faqList &&
-        faqList
+      {filteredFaqList &&
+        filteredFaqList
           .sort((a, b) => a.id - b.id)
```

Comment:
```suggestion
      {filteredFaqList?
        .sort((a, b) => a.id - b.id)
```
옵셔널 체이닝 문법을 쓰면 더 간단해질 것 같아요